src/components/command-bar/commands/search/components/dialog-body/index.tsx

```diff
@@ -69,47 +61,25 @@ const SearchCommandBarDialogBodyContent = ({
 		return generateSuggestedPages(currentProductTag?.id as ProductSlug)
 	}, [currentProductTag])
 
-	/**
-	 * Generate an object used to render all of the Tab elements to preselect the
-	 * Tab for the CurrentContentType.
-	 */
-	const tabsBySearchableContentType = useMemo<
```

Comment:
I've split this out to a separate file, which now exports `tabContentByType` object very similar to this `tabsBySearchableContentType`.

One notable difference is `tabContentByType` uses a `renderContent` function rather than a `content` slot, so that we can pass in the changing-at-render-time `currentProductTag` and `suggestedPages`. And `renderContent` also lets us pass in the revised `noResultsMessageSlot`, where the real substance of this PR happens.